src/player_controller.c

```diff
@@ -3428,124 +3427,124 @@ void func_80031F48(Player *player, f32 arg1) {
     }
 }
 
-void func_80032050(Player *player) {
-    s32 var_v0;
+void detect_triple_b_combo_b_released(Player *player) {
```

Comment:
How about:

detect_triple_b_combo_released


The idea is that the whole combo is pressed/released.

I think _b_released is somewhat repetitive. The function comment could always describe more thoroughly what's going on if the function name isn't descriptive enough.

Reply:
detect_triple_b_combo_released can let think of releasing the combo and not the button b